src/SystemProperties.php

```diff
@@ -186,6 +201,15 @@ public function jsonSerialize()
                 ],
             ];
         }
+        if (null !== $this->environment) {
```

Comment:
I'd avoid adding a workaround for environments unless strictly necessary

Reply:
Not a workaround, that's how everything is handled in `SystemProperties`